src/components/pages/observation/list/views/stats/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { SimpleGrid } from "@chakra-ui/react";
+import useObservationFilter from "@components/pages/observation/common/use-observation-filter";
+import React from "react";
+
+import SpeciesGroups from "./SpeciesGroups";
+import StatesDistribution from "./StatesDistribution";
+
+export default function StatsView() {
+  const { observationData, speciesGroup, filter }: any = useObservationFilter();
+
+  return (
+    <div>
```

Comment:
please remove this unnecessary `<div/>` wrapping since `<SimpleGrid/>` is just a single component